src/app/api/sumupPayment/sumup.api.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios"
+
+type response = {
+  id: string,
+  status: string
+}
+
+const pay_to_email = process.env.PAY_TO_EMAIL
+export async function getCheckoutId(amount: string) {
+  const referenceId = Math.floor(100000 + Math.random() * 900000)
+  try {
+    const apiUrl = "https://api.sumup.com/v0.1/checkouts"
+
+    const response = await fetch(apiUrl, { // Ensure the path is correct based on your file structure
+      method: "POST",
+      headers: {
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+        "Authorization": "Bearer sup_sk_om3EX5ZZhebf3h5esIYDaaNi8YT1I0ZKb"
```

Comment:
Again same mistake, we are putting credentials into github history

What is more, the final key will be different to the testing key depending on the environment.